shell/packages/sandstorm-db/db.js

```diff
@@ -368,6 +368,14 @@ ApiTokens = new Mongo.Collection("apiTokens");
 //                  database when that happens. This field contains the boolean true to signify that
 //                  it has been set.
 //       ipInterface: Ditto IpNetwork, except it's an IpInterface.
+//       emailVerifier: An EmailVerifier capability that is implemented by the frontend. The
+//                      value is an object containing the field `services`, which itself is a
+//                      list of names of identity providers that are trusted to verify addresses.
+//                      If `services` is omitted or falsy, all configured identity providers are
```

Comment:
I agree that naming would be slightly better but the difference isn't enough that I want to do the search-and-replace. :) (Wish Javascript had refactoring IDEs...)